regulation-worker/internal/delete/batch/batch.go

```diff
@@ -31,64 +28,28 @@ import (
 
 var (
 	pkgLogger             = logger.NewLogger().Child("batch")
-	regexRequiredSuffix   = regexp.MustCompile(".json.gz$")
 	StatusTrackerFileName = "rudderDeleteTracker.txt"
-	supportedDestinations = []string{"S3"}
+	supportedDestinations = []string{"S3", "S3_DATALAKE"}
 )
 
 const listMaxItem int64 = 1000
```

Comment:
I believe since we have defined `FilesLimit` in `BatchManager` struct, we don't need this anymore.

Reply:
I'll fix it.